app/models/merchant.rb

```diff
@@ -1,6 +1,8 @@
 class Merchant <ApplicationRecord
   has_many :items, dependent: :destroy
   has_many :item_orders, through: :items
+  has_many :orders, through: :item_orders
```

Comment:
This line is like what Brian showed us we could do for us_03 on our last challenge. I used the relationship to make my life easier, which was awesome! Just thought I'd show you guys where things like this can be useful.

Reply:
Way to remember to use that here! Thank you for pointing it out.